app/interfaces/utils/Route.ts

```diff
@@ -26,6 +27,10 @@ interface IRouteConfig {
   };
 }
 
+interface IMiddleware {
```

Comment:
Why you're creating the interface again ? you can import the one you created ? `IMiddleware.ts`